tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -174,4 +176,297 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 		deletePodsAndWaitForVolsToDetach(ctx, client, pods, true)
 	})
 
+	/*
+		Fill LZT/EZT volume
+		Steps:
+		1. create SPBM policies with LZT, EZT volume allocation
+		2. Create SCs using policies created in step 1
+		3. Create a large PVCs using SCs created in step 2
+		4. Wait and verify the PVCs created in step 3 is bound
+		5. Create a pod using PVCs created in step 3 and wait for it to be Ready.
+		6. Fill the PVCs by running IO within the pod created in step 5
+		7. Delete pod created in step 5
+		8. Delete the PVCs created in step 3
+		9. Delete the SCs created in step 2
+		10. Deleted the SPBM policies created in step 1
+	*/
+	ginkgo.It("[csi-block-vanilla][csi-block-vanilla-parallelized] Fill LZT/EZT volume", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		sharedvmfsURL := os.Getenv(envSharedVMFSDatastoreURL)
+		if sharedvmfsURL == "" {
+			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
+		}
+
+		largeSize := os.Getenv(envDiskSizeLarge)
+		if sharedvmfsURL == "" {
```

Comment:
I think this should be largeSize == ""

Reply:
I fixed it for the other test case, missed here, thanks